common/common.go

```diff
@@ -433,15 +434,46 @@ func GetBlock(cache *BlockCache, height int) (*walletrpc.CompactBlock, error) {
 	return block, nil
 }
 
+// Filters out blocks that have spammy transactions
```

Comment:
If I'm reading the code correctly, this description is incorrect: this function filters out "spammy" transactions *from* a block, it doesn't filter out the block.

Reply:
I agree; I took the liberty of updating the description for that reason, and in general to make it more complete.